wolfcrypt/src/aes.c

```diff
@@ -251,7 +292,44 @@ block cipher mechanism that uses n-bit binary string parameter key with 128-bits
             return ret;
 #endif
 
-    #ifdef WOLFSSL_STM32_CUBEMX
+    #ifdef WOLFSSL_STM32U5_DHUK
+        ret = wolfSSL_CryptHwMutexLock();
+        if (ret != 0)
+            return ret;
+
+        /* Handle making use of wrapped key */
+        if (aes->devId == WOLFSSL_STM32U5_DHUK_WRAPPED_DEVID) {
+            CRYP_ConfigTypeDef Config = {0};
+
+            ret = wc_Stm32_Aes_UnWrap(aes, &hcryp, (const byte*)aes->key,
+                aes->keylen, NULL);
+            if (ret != HAL_OK) {
+                ret = WC_TIMEOUT_E;
```

Comment:
Same, should this error case be doing something different below? Maybe even logging or failing?